api_v3/lib/types/EntryServerNode/base/filters/KalturaEntryServerNodeFilter.php

```diff
@@ -21,12 +21,6 @@ protected function getCoreFilter()
 	 */
 	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
 	{
-		if ($this->entryIdEqual == null &&
```

Comment:
I remembered there was a reason why I did not add this :)
This service is only open for partner -5 so there is no reason to add this validation.